src/components/FeatureViews/Search/Search.js

```diff
@@ -54,6 +54,9 @@ const Search = (props) => {
         if(SearchQuery !== prevSearchQuery){
             api.setSearchQuery(prevSearchQuery);
         }
+        return () =>{
+            sessionStorage.setItem("SearchQuery",SearchQuery);
+        }
     },[]);
```

Comment:
Should this be run only once or whenever the query changes?

Reply:
only whenever the query is changed. But the query isn't changed in Redux until the entire word has been typed out.

Reply:
In this case `SearchQuery` should probably be inside the dependencies array.